spec/feature/user_story_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe 'User stories' do
+  subject(:takeaway) { Takeaway.new(menu) }
+  subject(:menu) { Menu.new }
+
+  #User story 1
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'returns a list of dishes' do
+    expect(takeaway.menu_klass).to respond_to(:list)
+  end
+
+  it 'returns an item from the list' do
+    expect(takeaway.menu_klass.list[:Biryani_rice]).to eq (3.50)
+  end
+
+  context 'when selecting an order' do
+    before do
+      takeaway.menu_klass.list
+    end
+
+    #User story 2
+    # As a customer
+    # So that I can order the meal I want
+    # I would like to be able to select some number of several available dishes
+    it 'returns number of choices' do
+      takeaway.select_order(:Lamb_curry, 2)
+      takeaway.select_order(:Chicken_curry, 3)
+      expect(takeaway.order.size).to eq 2
+    end
+
+    it 'returns number of dishes' do
+      takeaway.select_order(:Lamb_curry, 2)
+      takeaway.select_order(:Chicken_curry, 3)
+      expect(takeaway.total_of_dishes).to eq 5
+    end
+  end
+
+  context 'when finalising an order' do
+    it 'checks the sum for my total order' do
+      # As a customer
+      # So that I can verify that my order is correct
+      # I would like to check that the total I have been given matches the sum of
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)